governance/tools-transfer.md

```diff
@@ -12,12 +12,19 @@ The tomesh tools team is responsible for managing the following:
 - Server: All the files are hosted on the tomesh1 [Digital Ocean](https://www.digitalocean.com) droplet.
 - DNS: [hover](www.hover.com) is the DNS service provider. This service allows us to map tomesh.net to the IP address of the tomesh1 droplet.
 - Github: The [tomesh.net repository](https://github.com/tomeshnet/tomesh.net) hosts the website files.
+- Build A Node page is pulled from (prototype-cjdns-pi)[https://github.com/tomeshnet/prototype-cjdns-pi] README.md
 
 Webhook is configured in Github to ping the server on push. 
 The droplet pulls its content from the Github repository periodically and whenever it is pinged.  
 The droplet uses nginx and Let's Encrypt to serve content over HTTPS.  
 nginx decides what content to serve depending on what domain the request is coming from.
 
+#### Map on Website 
+- Map Uses GOOGLE MAP API 
+- Current Google API Key: AIzaSyDfjIeSSWeDLOYt7f_cHHG4GtgOowvQFBs
```

Comment:
Just to confirm there is no billing enabled for the account owning this key right?

Reply:
I created the key and google does not have my credit card number :)  i think payed subscriptions are when you hit north of 200,000 hits or something crazy like that.

Reply:
Yup since we are posting it here, wanted to make sure if someone takes it the worst case is we get failing requests and not a large bill.

Reply:
Thanks :)

Key is public anyway (in the html code)
Correction - key is good for  25,000 hits per day most we had in a day was 171.
I also added a referral condition on the key to only serve maps for *.tomesh.net